src/DssSpell.sol

```diff
@@ -49,82 +127,197 @@ contract DssSpellAction is DssAction {
+            require(D3MOracleLike(D3M_ORACLE).ilk() == ILK, "Oracle ilk mismatch");
 
-    function actions() public override {
-        // -------------------- Update RWA007, RWA008, RWA009 Legal Documents ---------------------
-        // https://forum.makerdao.com/t/nov-16-2022-executive-contents/18747
-        //
-        // Monetalis (RWA007-A) legal update doc
-        _updateDoc("RWA007-A", "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR");
-        // SG Forge OFH (RWA008-A) legal update doc
-        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
-        // HVB (RWA009-A) legal update doc
-        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
-
-        // RWA007-A autoline changes:
-        // - bump oracle price to 500m
-        // - increase DC to 500m
-        // - increase autoline `gap` to 100m
-        //
-        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RWA007-A",
-            500 * MILLION,
-            100 * MILLION,
-            1 weeks
-        );
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA007-A",
-             500 * MILLION * WAD
-        );
-        DssExecLib.updateCollateralPrice("RWA007-A");
-
-        // RETH-A autoline changes:
-        // - line 5m
-        // - gap 3m
-        // - ttl 8h
-        //
-        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
-        DssExecLib.setIlkAutoLineParameters(
-            "RETH-A",
-            5 * MILLION,
-            3 * MILLION,
-            8 hours
-        );
-
-        // -------------------- Changelog Update & housekeeping ---------------------
-
-        // - Change "RWA007_A_INPUT_CONDUIT_URN" to "RWA007_A_INPUT_CONDUIT"
-        // - Change "RWA007_A_INPUT_CONDUIT_JAR" to "RWA007_A_JAR_INPUT_CONDUIT"
-        //
-        DssExecLib.setChangelogAddress("RWA007_A_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_URN"));
-        DssExecLib.setChangelogAddress("RWA007_A_JAR_INPUT_CONDUIT", DssExecLib.getChangelogAddress("RWA007_A_INPUT_CONDUIT_JAR"));
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_URN");
-        ChainlogLike(DssExecLib.LOG).removeAddress("RWA007_A_INPUT_CONDUIT_JAR");
-
-        // Bump version
-        DssExecLib.setChangelogVersion("1.14.5");
+            DssExecLib.setContract(D3M_HUB, ILK, "pool", D3M_COMPOUND_POOL);
+            DssExecLib.setContract(D3M_HUB, ILK, "plan", D3M_COMPOUND_PLAN);
+            DssExecLib.setValue(D3M_HUB, ILK, "tau", 7 days);
+            DssExecLib.setContract(D3M_HUB, "vow", DssExecLib.vow());
+            DssExecLib.setContract(D3M_HUB, "end", DssExecLib.end());
+
+            DssExecLib.setAuthority(D3M_MOM, DssExecLib.getChangelogAddress("MCD_ADM"));
+
+            DssExecLib.setContract(D3M_COMPOUND_POOL, "king", address(this));
+
```

Comment:
For 'Add Aave V2 D3M', the following line was there:

<img width="515" alt="Auth" src="https://user-images.githubusercontent.com/103920908/205303939-4cf4d64c-49a8-4534-be17-c81657a0811b.png">

Perhaps there could be something for plan or hub; is there a reason we aren't doing this now?

Reply:
This was discussed above. No need for ESM anymore.

Reply:
Yeah old module worked differently. This new structure was prepared to avoid `.rely(esm)` inclusion (which requires a keeper doing a tx after ES).